drivers/nrf_radio_802154/CMakeLists.txt

```diff
@@ -54,37 +48,21 @@ zephyr_compile_definitions(
   # radio IRQ priority to 0, or the radio IRQ will ignore IRQ lock.
   NRF_802154_IRQ_PRIORITY=1
   NRF_802154_INTERNAL_RADIO_IRQ_HANDLING=0
+  NRF_802154_INTERNAL_SWI_IRQ_HANDLING=0
 
   # Radio driver shim layer uses raw api
   NRF_802154_USE_RAW_API=1
 
-  # Number of slots containing short addresses of nodes for which
-  # pending data is stored.
```

Comment:
Why are the comments removed ?

Reply:
Short story: I will be easier to maintain.
Long story: Because documentation for this define is in the nRF 802.15.4 Radio Driver itself. Here explanations "why" something is set to certain value is valuable. 
The other case is that switching between `xxxxx_FEATURE=0` to `xxxxx_FEATURE=1` requires additional effort in changing comment `# Disable feature xxxxx` to `# Enable feature xxxxx` or moving to other sections. 

Please compare `NRF_802154_CSMA_CA_ENABLED=1` and value of comment `# Enable CSMA/CA`, aren't they mostly self-explanatory?